.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: java
+
+jdk:
+  - oraclejdk8
+  - oraclejdk7
+  - openjdk7
+
+before_install:
+  - sudo apt-get update -qq
+  - sudo apt-get install -y byacc
+
+before_script:
+  - mkdir -p build dist doc
+  # Get the latest fregec.jar to compile the compiler
+  - curl -L -o fregec.jar https://github.com/Frege/frege/releases/download/3.22.324/frege3.22.367-g2737683.jar
+
+script:
+  - make runtime 'shadow-prelude' compiler && make fregec.jar
```

Comment:
https://github.com/Frege/frege/wiki/Getting-Started doesn't mention about `shadow-prelude` target. Isn't it needed?